### PR TITLE
Issue 316 delete admin user entity

### PR DESCRIPTION
**Pull request by kate-kyzyma**
### Fixed Issues (if relevant)

1. magento/magento-functional-tests-migration#316: Convert DeleteAdminUserEntityTest to MFTF 

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/User/Test/Mftf/ActionGroup/AssertImpossibleDeleteYourOwnAccountActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertImpossibleDeleteYourOwnAccountActionGroup">
```
Comment (rogyar) [inline app/code/Magento/User/Test/Mftf/ActionGroup/AssertImpossibleDeleteYourOwnAccountActionGroup.xml]: Please, use the following naming convention here `Assert<Admin|Storefront><Functionality>ActionGroup`. So, need to add the area definition after the `Assert` keyword.

Thank you.

File: app/code/Magento/User/Test/Mftf/ActionGroup/AssertUserInGridActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        </arguments>
+        <amOnPage url="{{AdminUsersPage.url}}" stepKey="navigateToUsersGrid"/>
+        <click selector="{{AdminUserGridSection.resetButton}}" stepKey="resetGridFilter"/>
+        <waitForPageLoad stepKey="waitForFiltersReset"/>
+        <fillField selector="{{AdminUserGridSection.usernameFilterTextField}}" userInput="{{userName}}" stepKey="enterUserName"/>
+        <click selector="{{AdminUserGridSection.searchButton}}" stepKey="clickSearch"/>
+        <waitForPageLoad stepKey="waitForGridToLoad" time="15"/>
+        <see selector="{{AdminUserGridSection.usernameInFirstRow}}" userInput="{{userName}}" stepKey="seeUser"/>
+    </actionGroup>
+</actionGroups>
```
Comment (rogyar) [inline app/code/Magento/User/Test/Mftf/ActionGroup/AssertUserInGridActionGroup.xml]: Please, use an empty line at the very end of every file.

File: app/code/Magento/User/Test/Mftf/ActionGroup/AssertImpossibleDeleteYourOwnAccountActionGroup.xml (hunk 1 above)
Comment (rogyar) [reply app/code/Magento/User/Test/Mftf/ActionGroup/AssertImpossibleDeleteYourOwnAccountActionGroup.xml]: The same suggestion is fair for all action groups in the scope of current PR.

File: app/code/Magento/User/Test/Mftf/Test/AdminDeleteAdminUserEntityTest.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (rogyar) [inline app/code/Magento/User/Test/Mftf/Test/AdminDeleteAdminUserEntityTest.xml]: I would suggest splitting this big test onto two separate tests in the same manner as we had for the MTF coverage. So, in the result we will have two separate tests (files):

- Test of admin own account removal
- Test of a successful admin account removal

**Comment by kate-kyzyma** [comment]
Hello, @rogyar
Thank you for reviewing my changes. I have modified my test a little bit. Could you please check it?

Thanks.

**Pull request merged**